src/modules/core/components/MembersList/MembersListItem.tsx

```diff
@@ -110,21 +112,27 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
             size="s"
             colonyAddress={colonyAddress}
             address={walletAddress}
-            user={user}
+            user={userProfile}
             showInfo={!onRowClick || showUserInfo}
             domainId={domainId}
             notSet={false}
           />
         </div>
         <div className={styles.usernameSection}>
-          {displayName && (
-            <span className={styles.displayName} title={displayName}>
-              {displayName}
+          {userProfile.profile.displayName && (
+            <span
+              className={styles.displayName}
+              title={userProfile.profile.displayName}
+            >
+              {userProfile.profile.displayName}
             </span>
           )}
-          {username && (
+          {userProfile.profile.username && (
             <span className={styles.username}>
-              <UserMention hasLink={false} username={username} />
+              <UserMention
+                hasLink={false}
+                username={userProfile.profile.username}
```

Comment:
i feel that it would be better to destruct this object once at the top
{username, displayName} = userProfile.profile

Reply:
Sure